pkg/graph/schema.graphql

```diff
@@ -1489,7 +1482,8 @@ type Query {
   operationalNeed(id: UUID!): OperationalNeed!
   auditChanges(tableName: String!, primaryKey: UUID!): [AuditChange!]!
   possibleOperationalNeeds: [PossibleOperationalNeed!]!
-  planDocumentSolutionLinks(modelPlanID: UUID!): [PlanDocumentSolutionLink!]!
+  possibleOperationalSolutions: [PossibleOperationalSolution!]!
+  planDocumentSolutionLinks(solutionID: UUID!): [PlanDocumentSolutionLink!]!

```

Comment:
I'm not sure `planDocumentSolutionLinks` is needed on the query directly, if we are returning documents from the solution object.

Reply:
Hmm, at a glance I agree. This was from before we made the modification to add documents[] to the operational solution. Let me think about it a little then probably I'll remove it.